test/library/gui/dwarfmode.lua

```diff
@@ -64,4 +64,10 @@ function test.enterSidebarMode()
     guidm.enterSidebarMode(df.ui_sidebar_mode.Default)
     expect.eq(df.ui_sidebar_mode.Default, df.global.ui.main.mode)
     expect.eq('dwarfmode/Default', dfhack.gui.getCurFocus(true))
+
+    -- verify that all supported modes lead where we say they'll go
+    for k,v in pairs(guidm.SIDEBAR_MODE_KEYS) do
+        guidm.enterSidebarMode(k)
+        expect.eq(k, df.global.ui.main.mode)
+    end
```

Comment:
Should this switch back to `Default` when done? Or is that handled by ci/test.lua?

Reply:
it's handled by `ci/test.lua`, but I added a mode switch back to default so `ci/test.lua` doesn't have to report that it's autocorrecting anything.